### PR TITLE
fix(style and agile guides): Add changes for new review workflow

### PR DESCRIPTION
**Pull request by rhetoric101**
Here are some changes to reflect the new review workflow.

### CODE REVIEW

File: src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx [hunk 1]
```diff
@@ -47,42 +47,45 @@ The docs board has the following columns:
...
       </td>
       <td>
-        Like our Needs Peer Edit column in Jira: A writer has requested a review of their PR. Review their PR in GitHub and leave comments. 
-        
-        **Whoever takes it: assign yourself as Reviewer.**
+        When you pick up a PR from **Needs review**:
+        1. Make sure your name is listed as a reviewer.
+        2. Move the PR into this column while you review it.
+        2. Review the PR in GitHub and leave comments.
+        3. When you are finished with the review, move it to ** Waiting on technical writer to merge**.
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx:88]: Careful with the **, there's a space before Waiting.

File: src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx [hunk 2]
```diff
@@ -47,42 +47,45 @@ The docs board has the following columns:
...
       </td>
     </tr>
     <tr>
       <td>
-        **In review (Hero or any TW)**
+        **In progress**
       </td>
       <td>
-        Drag PRs to this column when they are being reviewed. This shows who is reviewing and what is being reviewed, so two writers don’t mistakenly work on the same PR.
+        Drag PRs to this column when you are working on them. Make sure your face is on the PRs so other writers don’t mistakenly work on it.
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx:61]: Them - PRs is plural.

File: src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx [hunk 3]
```diff
@@ -15,28 +15,42 @@ A high-altitude view of the `What's new` process looks like this:
...
   ```
-* If the post refers to any images, make sure they are not links to images hosted on other sites. Images files must reside in our repository.
-* Check if the post has the `isFeatured: true` in the front matter. If so, this means that the post will be pinned to the top of the product UI. You should remove the corresponding flag in the previously featured post so only one post is featured.
-* After you publish the post, confirm that it shows up in both the docs site and in the product UI.
-* For posts that are to be published in the future:
-  * Generally, the hero on duty takes responsibility for the post on the day it needs to be published. 
+3. If the post refers to any images, make sure they are not links to images hosted on other sites. Images files must reside in our repository.
+4. Check if the post has the `isFeatured: true` in the front matter. If so, this means that the post will be pinned to the top of the product UI. You should remove the corresponding flag in the previously featured post so only one post is featured.
+
+### Publish the post
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx:46]: Anchor

File: src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx [hunk 4]
```diff
@@ -15,28 +15,42 @@ A high-altitude view of the `What's new` process looks like this:
...
+### The GitHub Board [#the-board]
+
+1. In the GitHub projects board, move the post from the triage column to the **Scheduled work** column. The post will remain there until the post is actually published.
+2. Insert the publishing date and time window into the PR title so it’s easy to recognize. Our [instructions](src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx) tell contributors that we offer these publishing windows: morning, mid-day, or evening. You may need to ask if they want this posted early to ensure it is available on time.
+
+### What to review [#the-review]
+
+1. Review the doc using our main style guide rules. If you see major issues, work with the contributor (or the product marketing reviewer); otherwise, you can just make the changes yourself.
+2. Confirm that the URLs to the docs site are absolute:
+  <Callout variant="important">
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx:36]: Not sure this requires a callout.

**Review by urbiz-nr — DISMISSED** [review]
Looks good, though I left a few comments - check them out before merging! Thanks for updating this!

File: src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx:88]: @urbiz-nr, good catch! I'll fix this!

File: src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx:61]: @urbiz-nr, thanks! I'll fix it!

File: src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx:46]: @urbiz-nr, I"m not sure what "Anchor" means in this context. Were you suggesting I add one? If so, where?

Comment (urbiz-nr) [reply src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx:45]: In the header! As in [#publish-post] @rhetoric101.

Comment (rhetoric101) [reply src/content/docs/style-guide/writing-docs/article-templates/whats-new-tech-writer-tasks.mdx:45]: @urbiz-nr, I finally see you were pointing out a missing anchor in the header! I'll fix that! Thanks!

**Pull request merged**